### PR TITLE
Update Japanese text to 5.0.1

### PR DESCRIPTION
**Pull request by SentKurokawa**
Update to 5.0.1
Fix Mistranslation

### CODE REVIEW

**Comment by ZhenjaMax** [comment]
Please open the game and, close and send `Database.log`. 
(path to it in guide)

**Comment by SentKurokawa** [comment]
[Database.log](https://github.com/CivilizationVIBetterBalancedGame/BetterBalancedGame/files/9673598/Database.log)

File: lang/japanese.xml [hunk 1]
```diff
@@ -1,214 +1,124 @@
...
+    </Row>
+    <Row Tag="LOC_BBG_TEDDY_COMBAT_BONUS_IN_MEDIEVAL" Language="ja_JP">
+      <Text>中世時代に到達していることにより [ICON_Strength] 戦闘力+3(ルーズベルトの効果)</Text>
+    </Row>
+    <Row Tag="LOC_BBG_TEDDY_COMBAT_BONUS_IN_RENAISSANCE" Language="ja_JP">
+      <Text>ルネサンス時代に到達していることにより [ICON_Strength] 戦闘力+4(ルーズベルトの効果)</Text>
+    </Row>
+    <Row Tag="LOC_BBG_TEDDY_COMBAT_BONUS_IN_INDUSTRIAL" Language="ja_JP">
+      <Text>産業時代に到達していることにより [ICON_Strength] 戦闘力+5(ルーズベルトの効果)</Text>
+    </Row>
```
Comment (ZhenjaMax) [inline lang/japanese.xml:38]: Please check indents from the start of line.
It doesn't break file and not necessary now, but it doesn't according to file style and looks strange for others.

File: lang/japanese.xml [hunk 2]
```diff
@@ -315,468 +328,1105 @@
...
+		<!-- == NUBIA == -->
+		<!-- = civilization ability = -->
+		<Replace Tag="LOC_TRAIT_CIVILIZATION_TA_SETI_DESCRIPTION" Language="ja_JP">
+			<Text>長距離戦闘ユニットへの[ICON_Production]生産力+30%。すべての長距離戦闘ユニットは戦闘経験値+50%。戦略資源の鉱山の生産力[ICON_Production]+1。ボーナス資源と高級資源の鉱山のゴールド [ICON_Gold] +2。</Text>
+		</Replace>
+		<!-- = unique improvement = -->
+		<Replace Tag="LOC_IMPROVEMENT_PYRAMID_DESCRIPTION" Language="ja_JP">
+			<Text>
+        石工術の技術によって解禁される改善。草原、平原、砂漠、丘陵の砂漠あるいは氾濫原にしか建設できない。[ICON_FOOD]食料+1、[ICON_Faith]信仰力+2。都心に隣接している場合[ICON_Food]食料+2。隣接する区域によって追加の産出を得る。[NEWLINE]キャンパスに隣接するごとに、[ICON_SCIENCE]科学力+2。[NEWLINE]工業区域に隣接するごとに、[ICON_PRODUCTION]生産力+2。[NEWLINE]劇場区域に隣接するごとに、[ICON_CULTURE]文化力+2。[NEWLINE]聖地に隣接するごとに、[ICON_FAITH]信仰力+2。[NEWLINE]商業区域や港に隣接するごとに、[ICON_GOLD]ゴールド+2。[NEWLINE]他のヌビアのピラミッドの隣には建設できない。</Text>
+		</Replace>
```
Comment (ZhenjaMax) [inline lang/japanese.xml:729]: Please check tags - <Text>words</Text> should be on same line.
Same as previous: It doesn't break file and not necessary now, but it doesn't according to file style.

**Comment by SentKurokawa** [comment]
I understand.
I will fix it in the next update.

**Pull request merged**